### PR TITLE
Pubsub snippets minor updates

### PR DESCRIPTION
**Pull request by jabubake**
Adding message handler in  subscriber snippet
Renaming constant project id, topic ids to be consistent across projects

### CODE REVIEW

File: google-cloud-examples/src/main/java/com/google/cloud/examples/pubsub/snippets/SubscriberSnippets.java [hunk 1]
```diff
@@ -86,8 +87,11 @@ private void createSubscriber() throws Exception {
     MessageReceiver receiver = new MessageReceiver() {
           @Override
           public void receiveMessage(PubsubMessage message, AckReplyConsumer consumer) {
-            // handle incoming message, then ack or nack the received message
-            // ...
+            // handle incoming message, then ack/nack the received message
+            System.out.println("Id : " + message.getMessageId());
+            // decode from base64 to view original data
+            System.out.println("Data : " +
+                new String(DatatypeConverter.parseBase64Binary(message.getData().toStringUtf8())));
```
Comment (pongad) [inline google-cloud-examples/src/main/java/com/google/cloud/examples/pubsub/snippets/SubscriberSnippets.java]: Why do we need to decode base64? Is the original data base64 encoded? Pubsub works OK with binary data, doesn't it?

Comment (jabubake) [inline google-cloud-examples/src/main/java/com/google/cloud/examples/pubsub/snippets/SubscriberSnippets.java]: When `PubSubMessage` gets constructed, we auto-encode the data to base64 : 
`PubsubMessage pubsubMessage = PubsubMessage.newBuilder().setData(data).build();` 
Should n't the user need to decode to process then ?

Comment (jabubake) [inline google-cloud-examples/src/main/java/com/google/cloud/examples/pubsub/snippets/SubscriberSnippets.java]: Uggh, you are right. I'm sorry I got this confused with handling messages over a push endpoint where they get received base64 encoded.

Comment (jabubake) [inline google-cloud-examples/src/main/java/com/google/cloud/examples/pubsub/snippets/SubscriberSnippets.java]: Updated. Thank you.

**Pull request merged**